### ISSUE TITLE
Support for the latest Wwise version (2021.1.3.7665)

### ISSUE THREAD

**Issue by blink7** [microsoft/ProjectAcoustics#66]
Seems it's currently not supported by the latest Wwise. Trying to install this plugin gives me the following message:
```
Error: Cannot install from D:\Downloads\MicrosoftProjectAcoustics.plugins.Unreal\Wwise\2019.1\ProjectAcoustics
The specified package is incompatible with the target.
```

**Comment by NoelCross** [comment]
blink7,

We are aware that the latest Project Acoustics release doesn't support Wwise 2021 at this time.  We'd like to update our plugin to support Wwise 2021's new object pipeline by the end of the year, but we haven' started on that work yet.

If this is a blocker for you, there are a couple of options:
1. Get access to the source by joining the Source Licensing program (https://docs.microsoft.com/en-us/gaming/acoustics/what-is-acoustics#source-access) which is an addendum to the GDK program
2. Use Wwise 2019 for now
3. Send me a note at noelc@microsoft.com requesting a one-off copy of a Wwise 2021 plugin (no object pipeline support) to temporarily unblock your work until we ship an official update with Wwise 2021 support.

Thanks,
Noel

**Comment by Yoerik** [comment]
Any update on this?

I'm trying to run the Project Acoustics Unreal/Wwise sample project:
https://docs.microsoft.com/en-us/gaming/acoustics/unreal-wwise-sample

On the website, it explicitly says Wwise 2021 needs to be integrated into the Unreal project in order to build soundbanks. However, the download link https://www.microsoft.com/en-us/download/details.aspx?id=58090 only provides the plugin for Wwise 2019.

Is the PA plugin for Wwise 2021 available somewhere?

Thanks,
Yoerik

**Comment by MikeChemi** [comment]
Hi Yoerik, sorry for the confusion. Our documentation update went live in this last week before our downloads were updated. We're expecting to have things ready any day now. We'll let you know when the new downloads are available.

**Comment by Yoerik** [comment]
That's awesome! Thanks for the quick follow-up!

**Comment by MikeChemi** [comment]
Wwise 2021 support is available in the 3.0 release, available now!
